image/image.go

```diff
@@ -122,6 +148,31 @@ type genericManifest interface {
 	GetLayers() []string
 }
 
+type descriptor struct {
+	MediaType string `json:"mediaType"`
+	Size      int64  `json:"size"`
+	Digest    string `json:"digest"`
+}
+
+type manifestSchema2 struct {
+	Config descriptor   `json:"config"`
+	Layers []descriptor `json:"layers"`
+}
+
+func (m *manifestSchema2) String() string {
+	// TODO: does it make sense to have string here?
```

Comment:
If nothing is using the method, remove it from the interface, and all implementations.

… actually, there is one use, in `LayersCommand`. This needs to be rethought somehow.  (FWIW I have no idea how `skopeo layers` is supposed to be used when the caller does not know what the name of the created directory is and has no way to figure that out.)